generators/client/templates/angular/package.json.ejs

```diff
@@ -67,14 +67,14 @@
     "browser-sync": "2.18.13",
     "browser-sync-webpack-plugin": "1.2.0",
     "codelyzer": "4.0.1",
-    "copy-webpack-plugin": "4.2.3",
+    "copy-webpack-plugin": "4.4.1",
     "css-loader": "0.28.4",
     "exports-loader": "0.6.4",
-    "extract-text-webpack-plugin": "3.0.2",
-    "file-loader": "1.1.5",
+    "extract-text-webpack-plugin": "4.0.0-alpha.0",
+    "file-loader": "1.1.6",
     "generator-jhipster": "<%= packagejs.version %>",
     "html-loader": "0.5.0",
-    "html-webpack-plugin": "2.30.1",
+    "html-webpack-plugin": "sendilkumarn/html-webpack-plugin",
```

Comment:
This plugin is not officially released, till then we can use this fork. I dont want to use the webpack-contrib fork, since any adhoc changes to that fork will have issues in our application.

Similarly tslint-loader is not yet released, but their master supports it. So took a fork and we will use it till it is officially released.